### PR TITLE
Add missing polish translation

### PR DESCRIPTION
**Pull request by axzx**
I am targeting this branch, because it's backwards compatible.

## Changelog

```markdown
### Added
- Added missing polish translation
```

### CODE REVIEW

File: src/Resources/translations/SonataAdminBundle.pl.xliff [hunk 1]
```diff
@@ -464,23 +464,23 @@
             </trans-unit>
             <trans-unit id="title_select_subclass">
                 <source>title_select_subclass</source>
-                <target>title_select_subclass</target>
+                <target>Wybierz typ obiektu</target>
```
Comment (wbloszyk) [inline src/Resources/translations/SonataAdminBundle.pl.xliff:467]:
```suggestion
                <target>Wybierz podtyp obiektu</target>
```

File: src/Resources/translations/SonataAdminBundle.pl.xliff [hunk 2]
```diff
@@ -464,23 +464,23 @@
...
             </trans-unit>
             <trans-unit id="no_subclass_available">
                 <source>no_subclass_available</source>
-                <target>no_subclass_available</target>
+                <target>Niedostępne typy obiektu</target>
```
Comment (wbloszyk) [inline src/Resources/translations/SonataAdminBundle.pl.xliff:471]:
```suggestion
                <target>Niedostępne podtypy obiektu</target>
```

Comment (wbloszyk) [reply src/Resources/translations/SonataAdminBundle.pl.xliff:471]: A może `Brak zdefiniowanych podtypów` ?

**Comment by greg0ire** [comment]
Thanks @axzx !

**Pull request merged**